modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-select/src/main/resources/META-INF/resources/select_search.js

```diff
@@ -0,0 +1,143 @@
+AUI.add(
```

Comment:
We have a file name convention for support classes. I know it's not the best, but we should keep it standardized. So, can you rename this file to select_search_support.js?